src/shared/lib/notification/notification.ts

```diff
@@ -0,0 +1,19 @@
+import { Messages } from 'entities/Task';
+
+export const getPermission = () => {
+    if ('Notification' in window) {
```

Comment:
not shure, why this chech is needed, could you please clarify?

Reply:
It was just in the example I saw, and I forgot to remove it.